docs/content/en/docs/administration/permissions/manage_dashboard_permissions.md

```diff
@@ -0,0 +1,34 @@
+---
+title: "manage_dashboard_permissions"
+linkTitle: "manage_dashboard_permissions"
+weight: 15
+no_list: true
+superheading: "catalog_permission."
+---
+
+
+
+``manage_dashboard_permissions(workspace_id: str, dashboard_id: str, permissions_for_assignee: List[Union[CatalogPermissionsForAssignee, CatalogPermissionsForAssigneeRule]] ) -> None``
+
+Provide managing dashboard permissions for user and user groups.
+
+{{% parameters-block  title="Parameters"%}}
+{{< parameter p_name="workspace_id" p_type="str" >}}
+Workspace identification string. e.g. "demo"
+{{< /parameter >}}
+{{< parameter p_name="dashboard_id" p_type="str" >}}
+Dashboard identification string. e.g. "campaign"
+{{< /parameter >}}
+{{< parameter p_name="permissions_for_assignee" p_type="List[CatalogPermissionsForAssignee]" >}}
```

Comment:
This does not cover `CatalogPermissionsForAssigneeRule` which can be used to specify an assignee rule (to assign to a set of users)